modules/web/js/ballerina/components/annotation-container.jsx

```diff
@@ -301,9 +302,9 @@ class AnnotationContainer extends React.Component {
      * @memberof AnnotationContainer
      */
     getAnnotationPackageNameSuggestions(value) {
+        const matches = [];
         const escapedValue = this.escapeRegexCharacters(value.trim());
-
-        const regex = new RegExp(`^${escapedValue}`, 'i');
+        const regex = new RegExp(escapedValue, 'i');
```

Comment:
Is this regex used to check a certain string contains in another string ?
If so, can use https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes .

Example: https://github.com/ballerinalang/composer/blob/master/modules/web/js/ballerina/components/utils/autosuggest-html.jsx#L156